content/applications/inventory_and_mrp/inventory/routes/concepts/stock_warehouses.rst

```diff
@@ -1,47 +1,99 @@
-locations of the main stock location of warehouse A + B. Go to the locations menu, and edit the main
-location of your two warehouses. Then, change their parent location to main location of warehouse
-A+B.
+To create and edit locations, go to :menuselection:`Inventory --> Configuration --> Locations`. All
+locations are listed here,including the stock location of the virtual warehouse that was created.
+Click into the stock location for the virtual warehouse created previously, and under the
+**Additional Information** tab, change the :guilabel:`Location Type` from *Internal Location* to
+*View*. This identifies this location as a virtual one used to create a hierarchical structure for
+a warehouse, aggregating its child locations. Products can *not* be stored in this
+:guilabel:`Location Type`.
 
-.. image:: stock_warehouses/ware2.png
+.. image:: stock_warehouses/stock-warehouses-location-types.png
    :align: center
+   :alt: Warehouse location types in location creation screen.
 
-Sell a product from the virtual warehouse
-=========================================
+Configure physical warehouse locations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Let’s say you have two products, one stored in warehouse A and one stored in warehouse B. Now, you
-can create a new quotation for one of each product. Go to other information and choose Warehouse A+B
-in the shipping information.
+Navigate back to the :guilabel:`Locations` overview via the breadcrumbs, then click into the first
+physical warehouse stock location. Under :guilabel:`Parent Location`, select the virtual warehouse
+from the drop-down menu. Navigate back to the :guilabel:`Locations` overview and repeat this step
+for the second physical warehouse stock location. Both locations are now child locations of the
+virtual warehouse parent location.
 
-.. image:: stock_warehouses/ware3.png
+.. tip::
+   To use a virtual parent location as the default warehouse for sales orders, each salesperson can
+   have the warheouse assigned to them from the drop-down menu next to
+   :guilabel:`Default Warehouse` on their employee form.
+
+.. image:: stock_warehouses/stock-warehouses-employee-form.png
    :align: center
+   :alt: Default warehouse location on employee form.
+
+Example flow: Sell a product from a virtual warehouse
+=====================================================
+
+To sell products from multiple warheouses using a virtual parent location, have at least *two*
+products and at least *two* warehouses configured, with at least one product in each warehouse.
+
+To create a new :abbr:`RFQ (request for quotation)`, navigate to the :guilabel:`Sales` app, and
+click :guilabel:`New`. Fill out the inforamtion on the new quotation, and add the two products
+stored in the two warehouses. Then, click the :guilabel:`Other Info` tab, and change the
+:guilabel:`Warehouse` to the virtual warehouse created previously. Once the quotation has been
+filled out, :guilabel:`Confirm`.
```

Comment:
To sell products from multiple warheouses using a virtual parent location, there must be at least *two*
products and at least *two* warehouses configured - with at least *one* product in each warehouse.

To create a new :abbr:`RFQ (request for quotation)`, navigate to the :guilabel:`Sales` app, and
click :guilabel:`New`. Fill out the inforamtion on the new quotation, and add the two products
stored in the two warehouses. 

Then, click the :guilabel:`Other Info` tab, and change the :guilabel:`Warehouse` to the virtual warehouse that was previously created. Once the quotation has been filled out, click :guilabel:`Confirm`.